### PR TITLE
AMSI Protection Providers Enumeration TTP

### PR DESCRIPTION
**Pull request by w0rk3r**
Based on https://github.com/GhostPack/Seatbelt/blob/master/Seatbelt/Commands/Windows/AMSIProvidersCommand.cs

### CODE REVIEW

File: discovery/221bc8bd-077f-41cb-bc14-f415442e7082.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+id: 221bc8bd-077f-41cb-bc14-f415442e7082
+name: AMSI Protection Providers Enumeration
+description: Enumerates Providers registered for AMSI (AntiMalware Scan Interface), which can be used to discover Security Software on the host
+metadata:
+  version: 1
+  authors:
+    - w0rk3r
+  tags:
+    - Seatbelt
```
Comment (khyberspache) [inline discovery/221bc8bd-077f-41cb-bc14-f415442e7082.yml:9]: Remove the tags for now

Comment (khyberspache) [inline discovery/221bc8bd-077f-41cb-bc14-f415442e7082.yml:9]: Remove the tags for now `tags: []`

**Review by khyberspache — CHANGES_REQUESTED** [review]
Small change, but approved

**Comment by w0rk3r** [comment]
Fixed

**Review by khyberspache — APPROVED** [review]

**Pull request merged**